nginx/config.go

```diff
@@ -201,7 +201,7 @@ http {
 			proxy_set_header Upgrade $http_upgrade;
 			proxy_set_header Connection $connection_upgrade;
 
-			{{ if $enforceHTTPS }}if ($access_scheme != "https") {
+			{{ if or $enforceHTTPS $appConfig.SSL.Enforce }}if ($access_scheme != "https") {
```

Comment:
That is correct and is intentional by design. The requirements are that if the router enforces something, it is not override-able by the user.

Precedence for this behaviour exists at https://github.com/deis/router/blob/master/nginx./config.go#L182